Add warning to README (#530)
mattermost/mattermost-docker: README.md

# Production Docker deployment for Mattermost

## WARNING:

> The current state of this repository does not work out of the box as the server version 5.31+ require Postgres versions of 10 or higher.

Reviewer: ```suggestion
The current state of this repository doesn't work out-of-the box since Mattermost server v5.31+ requires PostgreSQL versions of 10 or higher.
```

Reviewer 2: How do you feel about "PostgreSQL version 10" (-"of")?

> We're actively working on a fix and revamp of this repository. In the meantime please refer to these upgrade instructions: https://github.com/mattermost/mattermost-docker/issues/489#issuecomment-790277661

Reviewer: ```suggestion
We're actively working on a fix to this repository. Until then, please refer to these upgrade instructions: https://github.com/mattermost/mattermost-docker/issues/489#issuecomment-790277661
```

Reviewer 2: Should we put a period at the end of this?

> This project enables deployment of a Mattermost server in a multi-node production configuration using Docker.

Reviewer: ```suggestion
This project enables a deployment of a Mattermost server in a multi-node production configuration using Docker.
```
